composer.json

```diff
@@ -15,7 +15,7 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.5 || ^7.0 || ^7.1",
```

Comment:
this change is not needed, `^7.0` means `>=7.0,<8.0`

Reply:
sorry, I didn't know it

Reply:
Please remove the change on this line and I will be happy to merge your PR.